### DISCUSSION TITLE
How to send notification inside or outside ConversationHandler? 20.0a4

### DISCUSSION THREAD

**Discussion by p17m0** [python-telegram-bot/python-telegram-bot#3254] [Q&A] [answered]
I have a  db and if smth changed how can i send a notification?
What do i need? What is the handler?

**Comment by Bibo-Joshi** [comment, marked as answer]
Maybe [this faq entry](https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-Asked-Questions#i-want-to-handle-updates-from-an-external-service-in-addition-to-the-telegram-updates-how-do-i-do-that) can help you
